handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -350,4 +350,16 @@ private void loadFilter(String filter, LoadBalancer lb) {
       }
     }
   }
+
+  private boolean isNotChanged(String p, String s) {
+    if (p == null && s == null) {
+      return this.policy == null && this.strategy == null;
+    } else if (p == null) {
+      return this.policy == null && s.equals(this.strategy);
+    } else if (s == null) {
+      return this.strategy == null && p.equals(policy);
+    } else {
+      return p.equals(policy) && s.contentEquals(strategy);
```

Comment:
why strategy need special contentEquals?

Reply:
maybe just implement a compare method like the below:
```
private static boolean isEqual(String str1, String str2) {
    return (str1 == null ? str2 == null : str1.equals(str2));
}
```
Then we can simply call `isEqual(policy, this.policy) && isEqual(strategy, this.strategy)` to get the result without so many if/else.

Reply:
@eric-lee-ltk  This method（isNotChanged()） only use for a judgement . i think there will not much change.so i think no need.

Reply:
@wujimin  oh! i make a mistake

Reply:
@laijianbin What you intend to do in the `isNotChanged` looks like just compare whether two variables are equal or not. You implementation looks too complicated!